runtime/doc/deprecated.txt

```diff
@@ -140,6 +140,7 @@ LSP FUNCTIONS
 - *vim.lsp.util.get_progress_messages()*	Use |vim.lsp.status()| or access
 						`progress` of |vim.lsp.client|
 - *vim.lsp.get_active_clients()*		Use |vim.lsp.get_clients()|
+- *vim.lsp.inlay_hint()*			Use |vim.lsp.inlay_hint.toggle()| instead.
```

Comment:
I think no. This will break for those using nightly, but there's no expectation of stability there, and no need to document pre-release breakages. `news.txt` and `deprecated.txt` should document things that change between releases, otherwise the signal-to-noise ratio becomes too low.

Reply:
removed